src/crimson/osd/osd_operation.h

```diff
@@ -36,6 +39,94 @@ static_assert(
   (sizeof(OP_NAMES)/sizeof(OP_NAMES[0])) ==
   static_cast<int>(OperationTypeCode::last_op));
 
+template <typename>
```

Comment:
better off moving `OperationRepeatSequencer` and friends into another file with some brief comments explaining what this class is for at the top of that file.